docs/design/chart_format.md

```diff
@@ -155,30 +137,23 @@ A definition file does not specify anything about individual pieces of the defin
 Fields:
 
 
-* Name: A human-readable name of the definition. This may contain UTF-8 alphanumeric text, together with dash and underscore characters.
-* Description: A human-readable description of the definition, not to exceed one paragraph of text. No text formatting is supported. A description may use any printable text characters allowed by the file format.
-* Version: A SemVer 2 semantic version of the chart (template files). Refer to the [instruction on semver.org](http://semver.org/).
-* Keywords: A list of human-readable keywords. A keyword may contain alphanumeric text and spaces.
-* Maintainers: A list of author objects, where an author object has two properties:
-   ** Name: Author name
-   ** Email: Author email (optional)
-* Source: A URL to the source code for this chart
-* Home: A URL to the home page for the project
-* Dependencies:
-   * Charts:
-      * Name: definition name
-      * Location: Full URL where definition can be found
-      * Version: SemVer version range (e.g. ~1.2, `^2`, `>=4.1.1,<5.0.0`). If no version is present, fall back to the Location field (if this has a long name or short name, version info may be present). If there is no version there, use the latest version as found in the destination repository.
-* Environment
-   * Name: Currently, always Kubernetes.
-   * Version: Version of Kubernetes required. SemVer version ranges are supported for this field (e.g. ~1.2).
-   * Extensions: List of extension names that are required (`extensions/v1beta1`). These should correspond to paths inside of the `apis/` endpoint on Kubernetes.
-   * APIGroups: List of API groups this chart requires in order to function.
+* name: A human-readable name of the definition. This may contain UTF-8 alphanumeric text, together with dash and underscore characters.
+* description: A human-readable description of the definition, not to exceed one paragraph of text. No text formatting is supported. A description may use any printable text characters allowed by the file format.
+* version: A SemVer 2 semantic version of the chart (template files). Refer to the [instruction on semver.org](http://semver.org/).
+* keywords: A list of human-readable keywords. A keyword may contain alphanumeric text and spaces.
+* maintainers: A list of author objects, where an author object has two properties:
+   * name: Author name
+   * email: Author email (optional)
+* source: A URL to the source code for this chart
+* home: A URL to the home page for the project
+* expander: Indicates how to process the contents of templates/
```

Comment:
This sub bullet is not displaying correctly in markdown. See the Markdown styling instructions at the link below. The problem is two asterisks. Should be one.